spid_cie_oidc/onboarding/templates/onboarding_entities.html

```diff
@@ -0,0 +1,37 @@
+<!-- Extends default Bootstrap Unical Italia template -->
+{% extends 'base.html' %}
+
+{% load static %}
+{% load i18n %}
+
+{% block page_title %}
+{% trans "OIDC Federation onboarding system" %}
+{% endblock page_title %}
+
+{% block container %}
+
+<script src="https://cdn.jsdelivr.net/npm/vue@2.6.14/dist/vue.js"></script>
```

Comment:
If we don't use Django-rest or any other framework for implementing OAS3 it would be better to avoid vue.js
in other words, we MUST use vue.js in the right way, otherwise it's better don't.

please remove this dependency and create a paginated object using django internals, here:
https://docs.djangoproject.com/en/4.0/topics/pagination/